types/slimerjs/index.d.ts

```diff
@@ -147,7 +147,7 @@ interface IWebPage {
     render(filename: string): void;
     renderBase64(format: string): string;
     sendEvent(mouseEventType: string, mouseX?: number, mouseY?: number, button?: string): Promise<void>;
-    sendEvent(keyboardEventType: string, key: string, null1?: void, null2?: void, modifier?: number): Promise<void>;
+    sendEvent(keyboardEventType: string, key: string, null1?: undefined, null2?: undefined, modifier?: number): Promise<void>;
```

Comment:
Huh, it actually says:
> Third and fourth argument are not taken account for keyboard events. Just give null for them.

Wierd.
By the way, string unions would be useful here, as the docs seem to provide a complete list of allowed strings. For keyboard events, that's

> ‘keyup’, ‘keypress’ or ‘keydown’